product_assortment_details/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Product Assortment Details",
```

Comment:
Details is a little bit too vague.

As you add a description, should be product_assortment_description

Reply:
sure, we can change that. Went for "details" as it's description + chatter.